src/rdkafka_interceptor.c

```diff
@@ -618,6 +624,33 @@ void rd_kafka_interceptors_on_thread_exit(rd_kafka_t *rk,
 }
 
 
+/**
+ * @brief Call interceptor on_broker_state_changed methods.
+ * @locality application thread calling rd_kafka_broker_add() or
```

Comment:
locality should just be "any" since it may also be called from internal librdkafka threads.